scripts/metadata-import.md

```diff
@@ -0,0 +1,40 @@
+# Metadata-Import script
+<!-- TOC -->
+* [Metadata-Import script](#metadata-import-script)
+  * [What it does?](#what-it-does)
+  * [When we use it?](#when-we-use-it)
+  * [How it Works?](#how-it-works)
+  * [Parameters](#parameters)
+  * [Conclusion/QuickGuide:](#conclusionquickguide)
+<!-- TOC -->
+## What it does?
+
+The Metadata-Import script is designed to import metadata after batch editing in DSpace.
```

Comment:
Again, probably should say "The `metadata-import` script..."     We also should note that it's only for CSV format.  So "...import metadata from a CSV after batch editing..."